fill/fill_station.cc

```diff
@@ -200,13 +212,46 @@ void RunServer(uint16_t port, std::shared_ptr<Server> server)
     server->Wait();
 }
 
+void setup_logging() {
+    // Create the log file
+    std::filesystem::create_directories("logs");
+
+    auto t = std::time(nullptr);
+    std::tm tm = *std::localtime(&t);
+
+    std::ostringstream oss;
+    oss << "logs/log_" << std::put_time(&tm, "%Y-%m-%d_%H-%M-%S") << ".txt";
```

Comment:
This might throw an error or not write logs if the `logs/` directory does not exist. That is the issue we ran into last year. Just test to make sure it works when `logs/` does not exist

Reply:
Good catch - that should already be covered a few lines above with `std::filesystem::create_directories("logs");`